todo_app/tests_e2e/test_selenium.py

```diff
@@ -0,0 +1,130 @@
+from todo_app.test_integrate import mock_get_lists
+from todo_app import app
+import pytest
+import os
+from threading import Thread
+from todo_app.item import Item
+import requests
+from dotenv import load_dotenv,find_dotenv
+import urllib3
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import time
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    board_id = create_trello_board()
+    os.environ['TRELLO_BOARD_ID'] = board_id
+    # list_id = create_trello_list(board_id)
+    # create_trello_card(list_id)
+    todolist_id = getToDoListid(board_id)
+    os.environ['TODO_LISTID'] = todolist_id
```

Comment:
Here you only override the Todo list, but this can lead to the app trying to move items to the done list specified in your `.env` file, which will either succeed (and our test is affecting our live board) or fail (and our test will fail) - we should also override the Done list

Reply:
Completed override of the doing and done list id